src/sasl/cyrsasl_scram.erl

```diff
@@ -161,23 +168,23 @@ unescape_username_attribute({EscapedUserName, ClientNonce}) ->
         UserName -> {ok, {UserName, ClientNonce}}
     end.
 
--spec get_scram_attributes(jid:username(), jid:lserver()) -> scram_att() | error().
-get_scram_attributes(UserName, LServer) ->
+-spec get_scram_attributes(jid:username(), jid:lserver(), any()) -> scram_att() | error().
```

Comment:
I'm only slightly uncomfortable at that `any()`, if anything, it could be an `atom()`. But if it could be an even more precise `sha()`, even better 😄

Reply:
Thanks @NelsonVides  for spotting this, will change so it will be more precise.